pkg/cmd/cmd.go

```diff
@@ -0,0 +1 @@
+package cmd
```

Comment:
this must be a hideous golang requirement?  (can't have empty directories that just contain sub-packages?)